.github/workflows/build-deploy-docs.yml

```diff
@@ -30,7 +30,6 @@ concurrency:
 
 jobs:
   build:
-    if: github.repository == 'future-proof-iot/RIOT-rs'
```

Comment:
Hm, I think I initially added this so forks don't deploy to fh-pages by default. Not sure whats best practice here

Reply:
Does the deploy fail if paged are disabled? (Would the forker get notifications fir that?)

Reply:
> It would indeed fail, and I did get a notification for that when it happened on my fork.

Well, that's why the skip should stay in IMO. I get annoyed when some unrelated workflows fail whenever I just update my fork's main branch to the upstream, that should be avoided.

We might at some point come up with sth configurable, e.g., "skip unless variable DEPLOY_DOCS==1", which every fork can then set together with enabling pages.

Reply:
No, unless this specific workflow can be disabled by default for forks but stay enabled for the main repository...